package.json

```diff
@@ -4,7 +4,7 @@
     "description": "Skeleton template for a Resin TypeScript project",
     "homepage": "https://github.com/resin-io/resin-typescript-skeleton#readme",
     "main": "lib/index.ts",
```

Comment:
Shouldn't this be `lib/index.js`? Otherwise anyone using the module will need to have a `.ts` handler registered in order to load the module